crypto/hd/fundraiser_test.go

```diff
@@ -83,7 +83,7 @@ func TestFundraiserCompatibility(t *testing.T) {
 
 		addr := pub.Address()
 		t.Logf("ADDR  \t%X %X\n", addrB, addr)
-		require.Equal(t, addr, crypto.Address(addrB), fmt.Sprintf("Expected addresses to match %d", i))
+		require.Equal(t, addr, cosmoscrypto.Address(addrB), fmt.Sprintf("Expected addresses to match %d", i))
```

Comment:
should we drop this dep, this is a merely a type for custom hexbytes. Asked years ago if we still need hexbytes but i couldnt remove it back then, maybe we can now?

Reply:
I'll take a look if we can drop that one 👍🏻

Reply:
ok gave it a second look and that type is being used in these places for example:
https://github.com/cosmos/cosmos-sdk/blob/b6e0a11e7981238cf7ff26c9c50a37a2d791fda3/crypto/keys/multisig/multisig.go#L38
https://github.com/cosmos/cosmos-sdk/blob/ca771f43f1ac6fabaf609d46ae9b3cce84731fa7/x/auth/types/credentials.go#L50

Not sure about deleting the type but I don't see the need to be an alias anymore, I can replace the usages to directly use the type `cosmoscrypto.Address`.
is that ok?